__tests__/app.test.js

```diff
@@ -144,6 +144,45 @@ describe("Test for GET API", () => {
                 })
         })
     })
+    describe("GET request to recive data on users", () => {
+        test("Status 200: Array of user data returned", () => {
+
+            const users =  [
+                {
+                  username: 'butter_bridge',
+                  name: 'jonny',
+                  avatar_url:
+                    'https://www.healthytherapies.com/wp-content/uploads/2016/06/Lime3.jpg'
+                },
+                {
+                  username: 'icellusedkars',
+                  name: 'sam',
+                  avatar_url: 'https://avatars2.githubusercontent.com/u/24604688?s=460&v=4'
+                },
+                {
+                  username: 'rogersop',
+                  name: 'paul',
+                  avatar_url: 'https://avatars2.githubusercontent.com/u/24394918?s=400&v=4'
+                },
+                {
+                  username: 'lurker',
+                  name: 'do_nothing',
+                  avatar_url:
+                    'https://www.golenbock.com/wp-content/uploads/2015/01/placeholder-user.png'
+                }
+              ]
+
+            return request(app)
+                .get("/api/users")
+                .expect(200)
+                .then(({ body }) => {
+                    expect(body).toMatchObject(users)
```

Comment:
Rather than expecting the body to match the entire object you've specified above, you can tidy this up by just expecting each item in the array to have the necessary properties 👍